dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -11,13 +11,65 @@
     = t(:slow_loading)
     %br
     %a{href: 'javascript: location.reload();'}= t(:try_reloading)
-  - if @level.game == Game.netsim
+
+-# Now define the dependencies for the level we are loading.
+-# These will be written at the end of the body tag so that we render as much
+-#   of the page as possible before running scripts, but so that browsers can
+-#   intelligently optimize downloading resources.
```

Comment:
Can the following code move to the controller?  If not, prefer a `:ruby` block over lots of `- code` statements in Haml.